### DISCUSSION TITLE
Turn Off SSL On Budibase

### DISCUSSION THREAD

**Discussion by herakim11** [Budibase/budibase#4935] [Help] [answered]
Is there any way to turn off SSL on Budibase if I want to use REST API?
Thank you 🙏🏽

**Comment by mjashanks** [comment, marked as answer]
If you are self-hosting, SSL is under your control - so you can turn it off as you need. If using Budibase Cloud, you cannot turn this off.

**Reply by crisacostaoficial** [reply]
How install?
